frontend/src/app/plans/account/page.tsx

```diff
@@ -0,0 +1,85 @@
+import Image from "next/image";
+import { notFound } from "next/navigation";
+import React from "react";
+
+import { Separator } from "@/components/ui/separator";
+import { createUsosService } from "@/lib/usos";
+
+const USER_STATUSES = {
+  0: "Nigdy nie byłeś studentem",
+  1: "Były Student",
+  2: "Aktywny Student",
+};
+
+const USER_SEX = {
+  M: "Mężczyzna",
+  K: "Kobieta",
+};
+
+export default async function ProfilePage() {
+  let profile;
+
+  try {
```

Comment:
Tu to samo z try, ale to moja preferencja, jakoś wolałbym tak jak wyżej pisałem, że zamiast wyjątkami rzucać to dawać jakieś message, ale nie wiem szczerze co lepsze